src/mcp/tools.cpp

```
@@ -0,0 +1,1267 @@
+
+    const auto& session =
+        engine.analyze(elf_path, section, program, type, opt_termination, opt_div_zero, opt_strict);
+
+    prevail::Label target_label = prevail::Label::entry;
+    bool is_error_query = false;
+
+    if (args.contains("pc")) {
+        const int pc = args["pc"].get<int>();
+        target_label = find_label_for_pc(session, pc);
+    } else {
+        const int error_index = args.value("error_index", 0);
```

Clang (14.0.6)

[warning] variable 'error_index' is not initialized

(cppcoreguidelines-init-variables)